client/src/design-system/components/editor/nodes/image-node.tsx

```diff
@@ -0,0 +1,257 @@
+import * as React from "react";
+import { JSX, Suspense } from "react";
+import type {
+  DOMConversionMap,
+  DOMConversionOutput,
+  DOMExportOutput,
+  EditorConfig,
+  LexicalEditor,
+  LexicalNode,
+  NodeKey,
+  SerializedEditor,
+  SerializedLexicalNode,
+  Spread,
+} from "lexical";
+import { $applyNodeReplacement, createEditor, DecoratorNode } from "lexical";
+
+const ImageComponent = React.lazy(() => import("../editor-ui/image-component"));
+
+export interface ImagePayload {
```

Comment:
on préfère utiliser des types plutôt que des interfaces
```suggestion
export type ImagePayload = {
```